Metadata/Driver/AnnotationDriver.php

```diff
@@ -175,7 +180,7 @@ public function loadMetadataForClass(\ReflectionClass $class)
                         $params[] = $this->convertReferenceValue($paramName, $annot->params[$paramName]);
                     }
 
-                    if (!$params) {
+                    if (!$params && !$annot instanceof Call) {
                         continue;
```

Comment:
@schmittjoh I think we should at least remove this check. Do you remember why you introduced it?